src/chart/animator/planner.cpp

```diff
@@ -106,17 +122,21 @@ void Planner::createPlan(const Gen::Plot &source,
 		addMorph(SectionId::color, step);
 		addMorph(SectionId::coordSystem, std::max(step, posDuration));
 
-		const auto &geomEasing =
-		    srcOpt->geometry == Gen::ShapeType::circle   ? in3
-		    : trgOpt->geometry == Gen::ShapeType::circle ? out3
-		    : srcOpt->geometry == Gen::ShapeType::line   ? in3
-		    : trgOpt->geometry == Gen::ShapeType::line   ? out3
-		                                                 : inOut5;
+		const ::Anim::Easing *geomEasing{&inOut5};
```

Comment:
if else

Reply:
The calng-tidy says that there are identical if-branches.